earth_enterprise/rpms/build.gradle

```diff
@@ -95,6 +71,21 @@ if (new File('build.properties').exists()) {
     }
 }
 
+// Look for command line parameters that specify how the packages are
+// to be built.  This can be either 'optimize' or 'debug'.
+// Option 'release' is the default.
+if ( project.hasProperty( 'optimize' ) ) {
+    scons_option = "optimize=1"
+}  // end check for option 'optimze'
+else if ( project.hasProperty( 'debug' ) ) {
+    scons_option = "release=0"
+}
```

Comment:
I think the original code was incorrect here...  We can keep `debug`  but we should map it to `scons_option = internal=1`.  The `release=0` is implied.  But we should also add we should add `internal` too because that is what scons really uses and have that map to the same thing of `scons_option = internal=1`.

Reply:
And we should have a check for `release` property and have that map to `scons_option = "release=1"`